scripts/kb_scripts/prepare_kb.py

```diff
@@ -33,6 +34,7 @@ def main(copy_kb_path: str):
                     prepared_repo_path.write(
                         os.path.join(os.path.split(copy_kb_path)[1], path.replace('../', '')) + '\n'
                     )
+            prepared_repo_path.write(os.path.join(os.path.split(copy_kb_path)[1], 'converted'))
```

Comment:
We already have prepared_kb filder in the root of ostis-web-platform. Why we need converted folder?